packages/core/src/App/Containers/app-notification-messages.jsx

```diff
@@ -100,6 +101,9 @@ const AppNotificationMessages = ({
 
     const notifications_limit = isMobile() ? max_display_notifications_mobile : max_display_notifications;
     const notifications_sublist = notifications.slice(0, notifications_limit);
+
+    if (!should_show_popups) return null;
```

Comment:
Do we need this extra prop(`should_show_popups`) to maintain the visibility of notifications?
Is it possible to check `notifications` length and show notifications based on it?

If we need an extra prop, can we rename the prop to `should_show_notifications`?

Reply:
@balakrishna-binary We need this prop in order to give us leverage to choose the exact moment when we want the pop-up notifications to appear regardless of their length. We also have an `is_notifications_visible` variable which is close by meaning to should_show_notifications but responsible for showing notifications that are viewed upon clicking the 'bell' icon in deriv-app header, unlike should_show_popups that is meant to define when to show pop-up notifications.
Would `should_notifications_pop_up` name be fine for you? Or Would you like me to change this variable name just inside this component while leaving it the same in NotificationStore? 😊
![Screenshot 2022-01-12 at 11 04 53](https://user-images.githubusercontent.com/87321516/149088191-34b2cfb9-4b51-4151-acad-d9bf18e35baa.png)
![Screenshot 2022-01-12 at 11 05 40](https://user-images.githubusercontent.com/87321516/149088217-fb557f56-1560-4c06-bce7-ce68503f581a.png)

Reply:
Okay, let's stick with the existing name.